### DISCUSSION TITLE
implenting clerk and turbo

### DISCUSSION THREAD

**Discussion by SamueleBarbiera** [mattddean/t3-app-router-edge-drizzle#3] [Ideas]
like this example https://github.com/clerkinc/t3-turbo-and-clerk

**Comment by mattddean** [comment]
Good idea, I think turborepo makes sense as an addition to this repo, and Clerk makes sense as a new repo where Auth.js is swapped out for Clerk.

**Reply by SamueleBarbiera** [reply]
let me know if u need help :) (it would be cool to implement the generation of zod type with drizzle-zod lib for the trpc input schemas)

**Reply by SamueleBarbiera** [reply]
i have also this example https://github.com/Unit-App/Unit-App (it uses expo as well)

**Reply by mattddean** [reply]
I was thinking turborepo would be good just to show the @trpc/next and drizzle auth.js adapter as separate packages, but feel free to fork or copy this repo and show how expo would fit in too!

**Reply by mattddean** [reply]
I tried adding turborepo [here](https://github.com/mattddean/t3-app-router-edge-drizzle/tree/feature/turborepo) but I'm getting the `Error: No QueryClient set, use QueryClientProvider to set one` error and I'm not sure why. I'm also having trouble convincing myself that it belongs in this repo. I think I'd like to keep this one a bit simpler. That being said, I still think a separate repo that integrates turborepo and clerk could be useful!
